Library/Formula/puddletag.rb

```diff
@@ -45,7 +38,40 @@ def install
     ENV.prepend_create_path "PYTHONPATH", HOMEBREW_PREFIX/"lib/python2.7/site-packages"
 
     bin.install Dir[libexec/"bin/*"]
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    bin.env_script_all_files(libexec/"bin", :PATH => HOMEBREW_PREFIX/"bin:$PATH", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    Pathname("Info.plist").write <<-EOS.undent
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+      <dict>
+        <key>CFBundleDevelopmentRegion</key>
+        <string>English</string>
+        <key>CFBundleExecutable</key>
+        <string>Puddletag</string>
+        <key>CFBundleIconFile</key>
+        <string>Puddletag</string>
+        <key>CFBundleIdentifier</key>
+        <string>com.github.keithgg.puddletag</string>
+        <key>CFBundleInfoDictionaryVersion</key>
+        <string>6.0</string>
+        <key>CFBundlePackageType</key>
+        <string>APPL</string>
+        <key>CFBundleVersion</key>
+        <string>1.0</string>
+        <key>CFBundleSignature</key>
+        <string>PUDDLE</string>
+      </dict>
+      </plist>
+    EOS
+
+    mkdir_p "Puddletag.app/Contents/MacOS"
+    mkdir_p "Puddletag.app/Contents/Resources"
+    cp buildpath/"Info.plist", "Puddletag.app/Contents"
+    system "sips", "-s", "format", "icns", buildpath/"puddletag.png", "--out", "Puddletag.app/Contents/Resources/Puddletag.icns"
+    cp prefix/"bin/puddletag", "Puddletag.app/Contents/MacOS/Puddletag"
+    chmod "+x", "Puddletag.app/Contents/MacOS/Puddletag"
+    prefix.install "Puddletag.app"
```

Comment:
Okay, I will.
Just some question on how you would accept it from upstream (sorry for being nasty):
Is it okay to give them a script containing a routine (bash script; the routine as is in the forumula) that can be run during the brew install process for making the bundle (which means acutally just creating predefined two files from the bash script routine, nothing more)?
Or is it necessary to push the whole bundle structure?

For clarity I could provide the contents of this script.

Using a script would be the easiest. Then the name of the executing script would just have to be patched/changed/set and the respective bash script would be more usable for upstream. Maintenance would be at upstream.

I hope I am not too nasty, but I think having your thoughts up front will provoke less effort for all.

Thanks in advance.

If there's another way to discuss that (private message, other means) pleaae let know. I won't bother the build system or so...